apps/portal/src/content/docs/components/alert-dialog.mdx

```diff
@@ -0,0 +1,341 @@
+      description: "Content to display",
+      required: true,
+      value: "ReactNode",
+    },
+    {
+      name: "className",
+      description: "Additional class names to apply to the alert dialog header",
+      required: false,
+      value: "string",
+    },
+  ]}
+/>
+
+### Footer
+
+The `Footer` component is used to define the footer section of the alert dialog.
+
+```typescript jsx
+<AlertDialog.Footer>
+  <AlertDialog.Cancel>Cancel</AlertDialog.Cancel>
+  <AlertDialog.Action>Confirm</AlertDialog.Action>
+</AlertDialog.Footer>
+```
+
+<DocsPage.PropsTable
+  props={[
+    {
+      name: "children",
+      description: "Footer content",
+      required: true,
+      value: "ReactNode",
+    },
+    {
+      name: "className",
+      description: "Additional class names to apply to the alert dialog footer",
+      required: false,
+      value: "string",
+    },
+  ]}
+/>
+
+### Title
+
+The `Title` component displays the title of the alert dialog.
+
+```typescript jsx
+<AlertDialog.Title>Alert Dialog Title</AlertDialog.Title>
+```
+
+<DocsPage.PropsTable
+  props={[
+    {
+      name: "children",
+      description: "Title content",
+      required: true,
+      value: "ReactNode",
+    },
+    {
+      name: "className",
+      description: "Additional class names to apply to the alert dialog title",
+      required: false,
+      value: "string",
+    },
+  ]}
+/>
+
+### Description
+
+The `Description` component displays the description of the alert dialog.
+
+```typescript jsx
+<AlertDialog.Description variant="secondary">
+  This is the alert dialog description.
+  <p>Alert dialog content goes here.</p>
+</AlertDialog.Description>
+```
+
+<DocsPage.PropsTable
+  props={[
+    {
+      name: "children",
+      description: "Description content",
+      required: true,
+      value: "ReactNode",
+    },
+    {
+      name: "className",
+      description:
+        "Additional class names to apply to the alert dialog description",
+      required: false,
+      value: "string",
+    },
+    {
+      name: "variant",
+      description: "Variant of the description",
+      required: false,
+      value: "'secondary' | 'quaternary'",
+      defaultValue: "'secondary'",
+    },
+  ]}
+/>
+
+### Action
+
+The `Action` component is used to define the action button of the alert dialog.
+
+```typescript jsx
+<AlertDialog.Action>Confirm</AlertDialog.Action>
+```
+
+<DocsPage.PropsTable
+  props={[
+    {
+      name: "children",
+      description: "Action button content",
+      required: true,
+      value: "ReactNode",
+    },
+    {
+      name: "className",
+      description:
+        "Additional class names to apply to the alert dialog action button",
+      required: false,
+      value: "string",
+    },
+  ]}
+/>
+
+### Cancel
+
+The `Cancel` component is used to define the cancel button of the alert dialog.
+
+```typescript jsx
+<AlertDialog.Cancel>Cancel</AlertDialog.Cancel>
+```
+
+<DocsPage.PropsTable
```

Comment:
Same here, we don't demonstrate how to trigger anything when someone cancels or confirms.